cmd: TestQuitQuitQuitGet failed

This test failed!

To configure my behavior, see [the Flaky Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/main/packages/flakybot).

If I'm commenting on this issue too often, add the `flakybot: quiet` label and
I will stop commenting.

---

commit: 1a7ea6fdae9aa807575888450d3790d6de742924
buildURL: https://github.com/GoogleCloudPlatform/alloydb-auth-proxy/actions/runs/12385164228
status: failed

=== RUN   TestQuitQuitQuitGet
2024/12/18 02:30:00 [proj.region.clust.inst] Listening on 127.0.0.1:5432
2024/12/18 02:30:00 The proxy has started successfully and is ready for new connections!
2024/12/18 02:30:00 Enabling quitquitquit endpoint at localhost:9192
2024/12/18 02:30:00 The proxy has encountered a terminal error: failed to start HTTP server: listen tcp 127.0.0.1:9192: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.
    root_test.go:1260: failed to dial endpoint: Get "http://localhost:9192/quitquitquit": dial tcp [::1]:9192: connectex: No connection could be made because the target machine actively refused it.
--- FAIL: TestQuitQuitQuitGet (11.03s)

--- a/cmd/root_test.go
+++ b/cmd/root_test.go
@@ -1238,7 +1238,7 @@ func TestQuitQuitQuitGet(t *testing.T) {
 	c := NewCommand(WithDialer(&spyDialer{}))
 	c.SilenceUsage = true
 	c.SilenceErrors = true
-	c.SetArgs([]string{"--quitquitquit", "--admin-port", "9192",
+	c.SetArgs([]string{"--quitquitquit", "--admin-port", "9193",
 		"projects/proj/locations/region/clusters/clust/instances/inst"})
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
@@ -1248,14 +1248,14 @@ func TestQuitQuitQuitGet(t *testing.T) {
 		err := c.ExecuteContext(ctx)
 		errCh <- err
 	}()
-	resp, err := tryDial("HEAD", "http://localhost:9192/quitquitquit")
+	resp, err := tryDial("HEAD", "http://localhost:9193/quitquitquit")
 	if err != nil {
 		t.Fatalf("failed to dial endpoint: %v", err)
 	}
 	if resp.StatusCode != http.StatusBadRequest {
 		t.Fatalf("expected a 400 status, got = %v", resp.StatusCode)
 	}
-	resp, err = tryDial("GET", "http://localhost:9192/quitquitquit")
+	resp, err = tryDial("GET", "http://localhost:9193/quitquitquit")
 	if err != nil {
 		t.Fatalf("failed to dial endpoint: %v", err)
 	}
@@ -1282,7 +1282,7 @@ func TestQuitQuitQuitWithErrors(t *testing.T) {
 	c.SilenceUsage = true
 	c.SilenceErrors = true
 	c.SetArgs([]string{
-		"--quitquitquit", "--admin-port", "9193",
+		"--quitquitquit", "--admin-port", "9194",
 		"projects/proj/locations/region/clusters/clust/instances/inst"})
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
@@ -1292,7 +1292,7 @@ func TestQuitQuitQuitWithErrors(t *testing.T) {
 		err := c.ExecuteContext(ctx)
 		errCh <- err
 	}()
-	resp, err := tryDial("POST", "http://localhost:9193/quitquitquit")
+	resp, err := tryDial("POST", "http://localhost:9194/quitquitquit")
 	if err != nil {
 		t.Fatalf("failed to dial endpoint: %v", err)
 	}

